app/views/static_pages/contributing/_hall_of_fame.erb

```diff
@@ -1,139 +1,30 @@
-    },
-    {
-      name: 'Mazin Power',
-      image: 'https://avatars1.githubusercontent.com/u/7075351?v=3&s=400',
-      github_url: 'https://github.com/muzfuz',
-    },
-    {
-      name: 'Ada Chiu',
-      image: 'https://avatars0.githubusercontent.com/u/13413986?v=3&s=400',
-      github_url: 'https://github.com/adachiu',
-    },
-    {
-      name: 'Kevin Mulhern',
-      image: 'https://avatars1.githubusercontent.com/u/7963776?v=3&s=460',
-      github_url: 'https://github.com/KevinMulhern',
-    },
-    {
-      name: 'Arun Kumar',
-      image: 'https://avatars0.githubusercontent.com/u/7039523?v=3&s=400',
-      github_url: 'https://github.com/arku',
-    },
-    {
-      name: 'Cody Loyd',
-      image: 'https://avatars0.githubusercontent.com/u/18574792?v=3&s=400',
-      github_url: 'https://github.com/codyloyd',
-    },
-    {
-      name: 'Cornelius Soon',
-      image: 'https://avatars2.githubusercontent.com/u/16495574?v=3&s=400',
-      github_url: 'https://github.com/csrail',
-    },
-    {
-      name: 'Rhys',
-      image: 'https://avatars3.githubusercontent.com/u/19835799?v=3&s=400',
-      github_url: 'https://github.com/105ron',
-    },
-    {
-      name: 'Austin',
-      image: 'https://avatars2.githubusercontent.com/u/12649577?v=3&s=400',
-      github_url: 'https://github.com/CouchofTomato',
-    },
-    {
-      name: 'Leo',
-      image: 'https://avatars2.githubusercontent.com/u/13614862?v=3&s=400',
-      github_url: 'https://github.com/leosoaivan',
-    },
-    {
-      name: 'Chad Kreutzer',
-      image: 'https://avatars1.githubusercontent.com/u/13308917?v=3&s=400',
-      github_url: 'https://github.com/ChadKreutzer',
-    },
-    {
-      name: 'Roberto Pando',
-      image: 'https://avatars1.githubusercontent.com/u/20132181?v=3&s=400',
-      github_url: 'https://github.com/RobPando',
-    },
-    {
-      name: 'Ryan',
-      image: 'https://avatars3.githubusercontent.com/u/20901017?v=3&s=400',
-      github_url: 'https://github.com/ryanford-frontend',
-    }
-  ]
-%>
+<section class="gradient">
+  <div class='page-container space-y-8'>
+    <h3 class="text-gold text-center font-bold text-2xl">Hall of Fame</h3>
 
-<div class="hall-of-fame gradient">
-  <h2 class="hall-of-fame__title text-2xl">Hall of Fame</h3>
-  <p class="hall-of-fame__text">
-    The Hall of Fame members are the hardworking devs who have built The Odin Project into the site that you see before you now.
-  </p>
+    <p class="text-center max-w-prose mx-auto">
+      The Hall of Fame members are the hardworking devs who have built
+      The Odin Project into the site that you see before you now.
+    </p>
 
-
-  <div class="container hall-of-fame__members">
-    <div class="row">
+    <div class="grid grid-cols-12 gap-6">
+      <% members = t('hall_of_fame') %>
       <% members.each do |member| %>
-        <div class="col-lg-2 col-md-3 col-sm-4 col-xs-6 hall-of-fame__member">
+        <div class="col-span-4 sm:col-span-3 md:col-span-2">
```

Comment:
The smallest and largest breakpoints don't appear to be kicking in, I tried a few different things but can't say for sure why they aren't working. It might be the gaps. But these breakpoints/col-spans work fine: 

```suggestion
        <div class="col-span-6 sm:col-span-3 lg:col-span-2">
```

Reply:
Ah bugger, these were both working before I pushed them, but now I'm checking them locally they're both broken there too. The classes don't seem to be being picked up, even when I roll back to a commit where I know they were working.

Strangely, at one point I couldn't get a margin utility to work (`mb-5`) but changing it to `mb-6` and it kicked in. 

Same thing seems to be happening here, classes just aren't being picked up at all. I'll try and have a play around with it and see if I can find the cause

Reply:
Ahh, I know what it is. The partial filename doesn't have a `.html` extension

Reply:
Ah, classic, always something dumb :lol: good eyes mate :pray:

I was getting there, I'd just managed to get the bundle to pick up the classes by pasting the component in the main template

Reply:
I wonder how long that had been hiding in there!

Reply:
> Maybe related to the Tailwind 3 upgrade?

Great point, you're absolutely right, It is related. TW3 uses JIT to load in classes whereas TW2 loaded everything in during development and purged any unused classes during the production build.